src/train.py

```diff
@@ -163,6 +198,7 @@ def train(
         save_strategy=config['training_settings']['save_strategy'],
         save_total_limit=config['training_settings']['save_total_limit'],
         output_dir=model_path,
+        logging_strategy='epoch'
```

Comment:
for consistency probably worth adding this to the config

Reply:
Added logging_strategy to `training_settings` in `train_config.yaml` and `Training Arguments` in `train.py`